test_rules_scala.sh

```diff
@@ -904,6 +904,26 @@ test_scala_import_fetch_sources() {
   assert_file_exists $bazel_out_external_guava_21/$srcjar_name
 }
 
+test_compilation_succeeds_with_plus_one_deps_on() {
+  bazel build --define plus_one_deps=true //test_expect_failure/plus_one_deps/internal_deps:a
+}
+test_compilation_fails_with_plus_one_deps_undefined() {
+  action_should_fail build //test_expect_failure/plus_one_deps/internal_deps:a
+}
+test_compilation_succeeds_with_plus_one_deps_on_for_external_deps() {
+  bazel build --define plus_one_deps=true //test_expect_failure/plus_one_deps/external_deps:a
+}
+test_compilation_succeeds_with_plus_one_deps_on_also_for_exports() {
+  bazel build --define plus_one_deps=true //test_expect_failure/plus_one_deps/exports_deps:a
+}
+test_plus_one_deps_only_works_for_java_info_targets() {
+  #for example doesn't break scala proto which depends on proto_library
+  bazel build --define plus_one_deps=true //test/proto:test_proto
```

Comment:
what do you think about configuring this in the toolchain?

I don't think we have a clear strategy about what is configured where. It seems this is a new direction, no?

Reply:
It’s a new direction for rules_scala but this is the current way for starlark rules to have command line configuration.

I’m good with moving it to the toolchain. Will do.